### DISCUSSION TITLE
區塊鏈遊戲，讓遊戲不再只是一種消遣，而是能實質上賺到錢的

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#490] [科技新知]
https://www.techbang.com/posts/93248-what-exactly-is-g
從小在玩遊戲時，我相信很多人都會想說能靠玩遊戲賺錢(可能不是每個人....但我是有這麼想過拉)，但長大後才發現要賺錢就是要變，而能達到那標準更是難上加難。但是我最近在學習區塊鏈這概念時，了解到了區塊鏈遊戲這塊，也就是所謂的gamefi，有感覺到或許小時候的想法能實現，雖然很難變成主業就是了。而能賺到錢我想主要也是多虧加密貨幣的興起與誕生，很多遊戲現在開始就能自己有貨幣，很多遊戲就是例如完成每日任務阿，與玩家對決等等就能拿遊戲幣，而這種遊戲幣就是真的能換真錢的那種!!!想當然幣的量一直增加肯定會掉價，因此遊戲方肯定也會有銷毀的機制。文章裡就有介紹甚麼是gamefi，play to earn的運作模式。話雖如此，這項技術還算很早期，許多方面都還算不成熟，因此很多遊戲創辦沒多久就消失了，我覺得主要是因為牽扯到貨幣經濟體有關讓遊戲運作更加複雜，也更深深考驗項目方的能力，期許在未來某一天，gamefi是能讓大眾普遍都能上手，風險也沒那麼高的!!

**Comment by sean1325418** [comment]
gamefi裡去中心化金融的元素可以解決原本若是遊戲倒了那遊戲資源就沒用了的問題，感覺這元素是裡面最重要的部分。

**Comment by ghost** [comment]
感覺伺服器的穩定度要先搞定的說XD

**Comment by bloodtrail0511** [comment]
老實說我不看好gamefi能經營得長久，為了利益而生的遊戲，本身就不會對遊戲性多加著墨，加入遊戲的人多半也都是把gamefi當成另一門工作罷了

**Comment by xkllkx** [comment]
目前的gamefi項目都還沒有很成熟，且都還是以快節奏營利為導向，會大幅限制可玩性，而且市面上充斥著一堆殺豬盤，要進場須警慎。

**Comment by d26488399** [comment]
這個機制看起來很理想，但其實沒有這呢好賺，畢竟有人賺錢就必定有人會虧錢。

**Comment by Yuhsin000** [comment]
可以賺錢聽起來很吸引人，但我不太會玩遊戲，進去應該是去送錢

**Comment by HaoFish123** [comment]
遊戲本來就是一種娛樂，是一種放鬆的管道，今天如果把它當成一種賺錢的工具，那好像失去了遊戲的本質。

**Comment by PoooYen** [comment]
那這樣算是電競選手嗎哈哈哈
